### DISCUSSION TITLE
감사합니다.

### DISCUSSION THREAD

**Discussion by chococookie1004** [changdoc/naver-vaccine-macro#86] [General]
잔여백신때문에 계속 안되서 10일정도 고생하다가 제작해주신 것 덕에 오늘 하루에 두개 잡았습니다.
이제 잔여백신 지옥에서 해방됐네요. 너무 감사합니다.

**Comment by changdocjjang** [comment]
> 잔여백신때문에 계속 안되서 10일정도 고생하다가 제작해주신 것 덕에 오늘 하루에 두개 잡았습니다.
> 이제 잔여백신 지옥에서 해방됐네요. 너무 감사합니다.

병원한개 크롬한개 2초인가요?

**Reply by chococookie1004** [reply]
저는 웨일에서 했고 웨일 1개에 1개 아이디 돌렸습니다. 다른 컴에 똑같이 웨일 1개에 다른 아이디 1개 돌리고요.
같은 병원은 안됐어요. 잔여가 1개씩 올라오다 보니까요

**Reply by taeheelee5** [reply]
한 컴퓨터에 웨일이랑 크롬에 각각 다른 계정으로 돌리신건가요?

**Reply by chococookie1004** [reply]
컴퓨터 2개로 아이디를 각자 돌렸습니다. 브라우저는 웨일 하나씩만 해놨어요. 블럭 먹을까봐...
